src/api/ror/GetRORsByName.ts

```diff
@@ -14,13 +14,22 @@ const GetRORsByName = async ({ query }: { query?: string }) => {
     let rors: Dict[] = [];
 
     if (query) {
+        /* Escape special characters in query string */
+        let escapedQuery: string = '';
+
+        for(let index = 0; index < query.length; index++) {
+            const character = query.at(index) as string;
+
+            escapedQuery = escapedQuery.concat(character?.replace(/[&/\\#,+()$~%.'":*?<>{}]/g, `\\${character}`));
```

Comment:
isn't it easier to have: `[^\w]`
This matches any character that isn't a number or a letter (a-zA-Z)

Reply:
Yay that also works!